src/hero/hero.macros.html

```diff
@@ -1,9 +1,18 @@
 {#
   * @overview Macros for hero and featured hero modules
+  * @module Hero
 #}
 
 {#
   * Renders a hero or featured hero module.
+  * @param {string} Heading text content
+  * @param {string} Subheading text content
+  * @param {string} Background image imgix path
+  * @param {boolean} Setting to display a large featured hero (optional, default: false)
+  * @param {string} Class name for imgix fluid targeting (optional, default: 'imgix-fluid--nocss')
```

Comment:
I was confused by "Imgix fluid targeting". This is the class that folks can use if they need to apply additional imgix js manipulations, right? I would find it much clearer to state something along the aforementioned lines if that's the case.